src/popup/components/AccountDetailsBase.vue

```diff
@@ -254,7 +244,7 @@ export default defineComponent({
   }
 
   .balance-info {
-    padding-top: calc(8px + env(safe-area-inset-top));
+    padding-top: 5em;
```

Comment:
I know that em/rem are more "modern" but we are using pixels heavily so I suggest to stick with this concept to avoid discrepencies.